build/dependencies.props

```diff
@@ -12,6 +12,7 @@
     <RedisVersion>1.2.3</RedisVersion>
     <TestSdkVersion>15.3.0-*</TestSdkVersion>
     <XunitVersion>2.3.0-beta2-*</XunitVersion>
+    <RxVersion>3.1.1</RxVersion>
```

Comment:
Shoot, I meant to wait to push this until I got confirmation from @Eilon, but I've already started the process with him.